civis/ml/_model.py

```diff
@@ -527,7 +502,7 @@ def _set_model_exception(fut):
             # If there's no metadata file
```

Comment:
Hi Stephen, I updated the name and took out the repeated callback attributes in the init. It seems to be working properly, but I had to move some of the other attributes being set in the init to above the `super().__init__()` since the `_set_job_exception` method relies on some of those properties being set.